tools/cdi-containerimage-server/import.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
```

Comment:
this file was only included for the proof of concept.  Should be regular http import container/code path

Reply:
It uses regular cdi-importer

Reply:
it should use the exact same code path as http import not call this script

Reply:
I stand corrected just missing some env vars

Reply:
Done